proxmox/virtual_environment_vm_types.go

```diff
@@ -478,7 +478,7 @@ type VirtualEnvironmentVMGetStatusResponseData struct {
 	Lock             *string     `json:"lock,omitempty"`
 	MemoryAllocation *int        `json:"maxmem,omitempty"`
 	Name             *string     `json:"name,omitempty"`
-	PID              *string     `json:"pid,omitempty"`
+	PID              *float64    `json:"pid,omitempty"`
```

Comment:
I would use `int` here, don't think PID can have a floating point.
```suggestion
	PID              *int        `json:"pid,omitempty"`
```

Reply:
there is a similar problem with `VMID` field below, would you mind update it to `int` as well?

Reply:
I've made both corrections, as requested.